src/test/java/de/tum/in/www1/artemis/util/DatabaseUtilService.java

```diff
@@ -94,6 +94,8 @@
 
     private static final Set<Authority> adminAuthorities = Set.of(userAuthority, tutorAuthority, editorAuthority, instructorAuthority, adminAuthority);
 
+    private static int dayCount = 1;
```

Comment:
Since it is static, If this value is mutated maybe it would be better to declare it inside the calling test method and pass it as an argument to `createBasicPost` method to prevent the risk of introducing unexpected behavior by having global state, what do you think?

Reply:
Thanks for your feedback. The method is called from 3 different locations and my intention was that createBasicPosts is called from anywhere the programmer likes and the posts created have different creation dates. If we update the code like suggested then the caller would have to handle this logic within each function that calls the mentioned method. 

In my opinion, the current implementation is adequate. However, we can change with respect to the severity of the risk.